frontend/src/Components/EventsCard.jsx

```diff
@@ -29,22 +30,41 @@ const EventsCard = (props) => {
                     break;
                 }
             }
+
             setLoggedVolunteerPartOfEvent(found);
             setLoggedVolunteerRequestAccepted(accepted);
         }
-        
+
         checkIfVolunteerSignedForEvent();
     }, [loggedUser, volunteersList])
 
+    const addHours = async (event_duration, v_id) => {
+        let vHours = await (await axios.get(`/api/time/hours/${v_id}`)).data.payload.banked_time;
```

Comment:
did you mean `let vHours = (await axios.get(`/api/time/hours/${v_id}`)).data.payload.banked_time;` ?

Reply:
Yes, this was a typo that is now fixed.